src/consumer/mod.rs

```diff
@@ -52,8 +52,9 @@ pub trait JobRunner: Send + Sync {
     /// A handler function that runs a job.
     fn run(&self, job: Job) -> Result<(), Self::Error>;
 }
+type BoxedJobRunner<E> = Box<dyn JobRunner<Error = E>>;
```

Comment:
I don't think we're using this type alias anywhere?

Reply:
The type alias is still used in the [unchanged `Consumer` struct](https://github.com/jonhoo/faktory-rs/pull/51/files#diff-c0b949907fe3d15f1fd945390ec35d975eb141f2f4a5724dcd82a9bf7470593aR196) (and it's builder):
```rust
pub struct Consumer<S, E>
where
    S: Read + Write,
{
    c: Client<S>,
    worker_states: Arc<Vec<Mutex<WorkerState>>>,
    callbacks: Arc<FnvHashMap<String, BoxedJobRunner<E>>>,
    terminated: bool,
}
```

Reply:
Ah, I missed that, thanks!